PlayCover/de.lproj/Localizable.strings

```diff
@@ -85,6 +86,7 @@
 
 "settings.picker.displayRefreshRate" = "Screen refresh rate";
 "settings.picker.screenSize" = "Display resolution";
+"settings.picker.ipadModel" = "Select iPad Pro model";
```

Comment:
I would recommend changing this locale to "settings.picker.iosDevice" or something similar since we may add iPhones under the same system in the future.